README.md

```diff
@@ -100,8 +100,10 @@ The following CMake options are supported:
   version of `clang` this option enables the use of the sanitizers documented
   [here](http://clang.llvm.org/docs/UsersManual.html#controlling-code-generation).
   This should only be used with a debug build. Disabled by default.
-* `SPIRV_WARN_EVERYTHING=OFF` - On UNIX platforms enable the `-Weverything`
-  compiler front end option, disabled by default.
+* `SPIRV_WARN_EVERYTHING=OFF` - On UNIX platforms enable more strict warnings,
```

Comment:
Actually I'd like to see these options to be represented as `SPIRV_WARN_EVERYTHING={ON|OFF}` and then have `{dis|en}abled by default` at the end of the sentence. Right now it reads like we only support `OFF`.